Branch-SDK-TestBed/src/io/branch/branchandroiddemo/MainActivity.java

```diff
@@ -337,6 +339,15 @@ public void onInitFinished(JSONObject referringParams,
     }
 
     @Override
+    protected void onResume() {
+        super.onResume();
+        new RegisterViewBuilder("my_test_content_id", "my_test_title", "my_test_desc", "http://my_content_url")
```

Comment:
can we give this the exact same pattern for URL creation? I'd like them to be as similar as possible, as it will mimic our content object in the future.